### PR TITLE
New PR to move all files in proper directories. Related issue: #9307

### LINKED ISSUE #9307: Adding new language - Uzbek (Cyrillic)
##### **Overview of the feature request**

It would be great to add a new Uzbek language based on Cyrillic alphabet as well. 

##### **Motivation for or Use Case**

Firstly, thanks to all contributors of JHipster, and also to a guy who helped  to add Uzbek - latin language. But there is a high demand on Uzbek - cyrillic as well, as most of our state documentations are written in Cyrillic alphabet.

### PR DESCRIPTION
**Pull request by khasanboyshoraimov**
Fix #9307

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by avdev4j** [comment]
why do you open many PR for this issue ? It seems that you have the commits from #9310 too.
You should create only one branch on you fork and open a Work In Progress PR here.

**Comment by khasanboyshoraimov** [comment]
I have added other files as well (.ejs), and moved all json files to correct directory.

**Comment by avdev4j** [comment]
I think you can simplify your process by using only one branch to do this and work on only one PR. you can start you PR with "WIP" tag to announce that this PR is not ready to merge and you are working on it.

By doing this you can avoid having more than one PR for the same issue. Of course you can split your PR too but in your case I think it's not recommended. We need the translation to be done in only one PR to be sure we have a working feature.

I advise you to work on only one branch, open a WIP PR, push your commits when they are ready, squash them if needed and do rebase too if needed.

It will be easier for you and for us to follow you work and what you have done 👍

**Comment by khasanboyshoraimov** [comment]
Previous file uploads are no longer needed, I would say. This PR is done, it contains all necessary i18n files. Thanks

**Comment by ruddell** [comment]
Nice work.  Please add the language to [the array here](https://github.com/jhipster/generator-jhipster/blob/07592aeb5c32d8bdd2d56f57255e7954550379b1/test/templates/all-languages/.yo-rc.json#L30) so that it will be tested

File: test/templates/all-languages/.yo-rc.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
       "th",
       "ua",
       "uz-lat",
+      "uz-cyr"
```
Comment (ruddell) [inline test/templates/all-languages/.yo-rc.json]: You need to add a comma after this line and the tests should pass

**Comment by avdev4j** [comment]
hi @khasanboyshoraimov 
so we have to close the #9307 ?

**Comment by khasanboyshoraimov** [comment]
Hello, @avdev4j . Yes, #9310 should be closed. Thank you so much

**Comment by jdubois** [comment]
LGTM I'm merging and add the documentation

**Pull request merged**